careers/devops.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2021-12-28
+status: Open
+---
+
+# Job Details
+
+## Responsibilities
+
+* Maintain GitLab and CI infrastructure for GHC
+* Improve CI
+* Expand the CI infrastructure to include base the base Haskell ecosystem
+* Support other Haskell libraries and tools, including HLS (Haskell Language Server)
+* Work with the Haskell Infrastructure Team
+* Support OS packaging of GHC and tools
+* Documentation
+* Improvements and expansion of the GHC performance dashboard
+
+## Qualifications
+
+* Demonstrated ability to manage and simplify complexity
+* Collaborate with a variety of stakeholders
+* Can juggle multiple, conflicting priorities
+* Excellent communication skills
+
+The Haskell Foundation is looking for a DevOps engineer to take the Haskell ecosystem, primarily the Glasgow Haskell Compiler (GHC, the Haskell compiler), to the next level of stability, reliability, and performance.
+
+This role spans many Haskell ecosystem projects to homogenize and improve the infrastructure used to build GHC and other core open source projects whose maintainers are interested, for instance HLS, Cabal, Stack, GHCup, core libraries, and more.
+
+The successful candidate will be in the middle of it all, collaborating with developers at the forefront of making Haskell a groundbreaking, influential, and top tier programming language. The individual will improve existing infrastructure, work on creating a shared CI and performance benchmarking system across the Haskell ecosystem, shorted CI turnaround times, and improve debugability of failures.
```

Comment:
```suggestion
The successful candidate will be in the middle of it all, collaborating with developers at the forefront of making Haskell a groundbreaking, influential, and top-tier programming language. The individual will improve existing infrastructure, work on creating a shared CI and performance benchmarking system across the Haskell ecosystem, shortened CI turnaround times, and improved ability to debug failures.
```

I feel like when I've seen "successful candidate" wording, the text is about what the candidate has done, not what the candidate will do.